src/software/field_tests/field_test_fixture.py

```diff
@@ -102,7 +102,7 @@ def time_provider(self):
         """Provide the current time in seconds since the epoch"""
 
         with self.timestamp_mutex:
-            return self.timestamp
+            return time.time()
```

Comment:
We don't need the lock anymore. 

Also, you can now remove `self.timestamp` and `self.ssl_wrapper_buffer`

Reply:
Are you sure you want to remove `self.timestamp` and `self.ssl_wrapper_buffer`. They are used else where:

I have removed instances of `self.timestamp` and `self.ssl_wrapper_buffer` within the context of `FieldTestRunner`. 

https://github.com/UBC-Thunderbots/Software/blob/02f6226a79ef7a44ecfb4aa56516a16bb58d3381/src/software/simulated_tests/simulated_test_fixture.py#L158-L160

Reply:
Oh didn't realize that its inherited. Don't remove it from the base class, just make `FieldTestFixture` not use the ssl_wrapper_buffer